### PR TITLE
fix(ui-tree-browser): fix TreeButton theme overrides

### PR DESCRIPTION
**Pull request by ke1k0**
Closes: INSTUI-3581

### CODE REVIEW

File: packages/ui-tree-browser/src/TreeBrowser/TreeNode/theme.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/**
+ TODO: this is a copy of the Button's theme, keep in sync!
```
Comment (HerrTopi) [inline packages/ui-tree-browser/src/TreeBrowser/TreeNode/theme.js:26]: I think you should add the sync warning to the Button's theme file as well.

File: packages/ui-tree-browser/src/TreeBrowser/TreeNode/__tests__/TreeNode.test.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ import {
 } from '@instructure/ui-test-utils'
 
 import { TreeNode } from '../index'
-import styles from '../../TreeButton/styles.css'
```
Comment (HerrTopi) [inline packages/ui-tree-browser/src/TreeBrowser/TreeNode/__tests__/TreeNode.test.js:35]: Do you know why it messes up the theming?

**Review by HerrTopi — APPROVED** [review]

File: packages/ui-tree-browser/src/TreeBrowser/TreeNode/__tests__/TreeNode.test.js (hunk 2 above)
Comment (ke1k0) [reply packages/ui-tree-browser/src/TreeBrowser/TreeNode/__tests__/TreeNode.test.js:35]: yeah, I forgot to change here, it broke the test, but I've fixed it :) (the classes are generated with hash, and it was looking for the buttons css hash instead of the node css hash)

File: packages/ui-tree-browser/src/TreeBrowser/TreeNode/styles.css [hunk 3]
```diff
@@ -0,0 +1,328 @@
+/* stylelint-disable no-descending-specificity, selector-max-class */
+
+/**
+  TODO: this is a copy of the Button's css, keep in sync!
```
Comment (HerrTopi) [inline packages/ui-tree-browser/src/TreeBrowser/TreeNode/styles.css:4]: I think you should add the sync warning to the Button's css file as well.

File: packages/ui-tree-browser/src/TreeBrowser/README.md [hunk 4]
```diff
@@ -137,358 +151,14 @@ class Example extends React.Component {
...
   }
 }
 
 render(<Example/>)
 ```
 
-### Customizing Icons
-
-All of the `<TreeBrowser>` icons are customizable. The following example sets custom icons for the expanded and collapsed state of the collections along with each of the items.
-
```
Comment (matyasf) [inline packages/ui-tree-browser/src/TreeBrowser/README.md:150]: Why did you delete so much from its Readme? These look like useful examples.

File: packages/ui-tree-browser/src/TreeBrowser/TreeNode/theme.js (hunk 1 above)
Comment (ke1k0) [reply packages/ui-tree-browser/src/TreeBrowser/TreeNode/theme.js:26]: done

File: packages/ui-tree-browser/src/TreeBrowser/TreeNode/styles.css (hunk 3 above)
Comment (ke1k0) [reply packages/ui-tree-browser/src/TreeBrowser/TreeNode/styles.css:4]: done

File: packages/ui-tree-browser/src/TreeBrowser/README.md (hunk 4 above)
Comment (ke1k0) [reply packages/ui-tree-browser/src/TreeBrowser/README.md]: Oh, my mistake, was using the Readme for testing and forgot to revert before the commit. Fixed!

**Review by matyasf — APPROVED** [review]
